traits/api.py

```diff
@@ -228,24 +228,13 @@
 try:
     from . import has_traits as has_traits
 
-    # ---------------------------------------------------------------------------
-    #  Patch the main traits module with the correct definition for the
-    #  ViewElements class:
-    #  NOTE: We do this in a try..except block because traits.ui depends on
-    #  the pyface module (part of the TraitsGUI package) which may not
-    #  necessarily be installed. Not having TraitsGUI means that the 'ui'
-    #  features of traits will not work.
-    # ---------------------------------------------------------------------------
-
-    from traitsui import view_elements
-
-    has_traits.ViewElements = view_elements.ViewElements
-
     # -------------------------------------------------------------------------------
     #  Patch the main traits module with the correct definition for the
     #  ViewElement and ViewSubElement class:
     # -------------------------------------------------------------------------------
 
-    has_traits.ViewElement = view_elements.ViewElement
+    from traitsui import view_element
```

Comment:
Can we fix the docstring above to remove mention of `ViewSubElement`?

Reply:
Yes, although that docstring is not long for this world - there will be a second PR to fix the `ViewElement` part of the problem.